### DISCUSSION TITLE
Adopt SpacyY Heavy Lifters (SYL)

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/SpaceYLifters#38] [General]

**Comment by ScottDB8** [comment]
Please adopt it! I love SpaceY Heavy Lifters!

**Comment by zer0Kerbal** [comment]
> Please adopt it! I love SpaceY Heavy Lifters!

@ScottDB8 - it is in the process of being adopted. Just want to do it justice and do it right!

**Comment by Ruedii** [comment]
Could you get the blocking bugs into the bug tracker so we can help?

**Comment by zer0Kerbal** [comment]
> Could you get the blocking bugs into the bug tracker so we can help?

it isn't bugs that is slowing me down, I had to create a process to recreate all the previous releases.

I finally have the process down, and it just will take 8-10 hours in one block to do it. I will see if I can get to it this week. 

After that - it is just my normal procedures to adopt..
